python/cudf_polars/cudf_polars/dsl/expressions/rolling.py

```diff
@@ -61,21 +59,21 @@ def to_request(
 
 
 class RollingWindow(Expr):
-    __slots__ = ("closed_window", "following", "orderby", "preceding")
-    _non_child = ("dtype", "preceding", "following", "closed_window", "orderby")
+    __slots__ = ("closed_window", "offset", "orderby", "period")
+    _non_child = ("dtype", "period", "offset", "closed_window", "orderby")
 
     def __init__(
         self,
         dtype: DataType,
-        preceding: pa.Scalar,
-        following: pa.Scalar,
+        offset: Duration,
+        period: Duration,
```

Comment:
@wence- - Does this seem like a reasonable way to make a `RollingWindow` object serializable? Note that these `preceding` and `following` arguments were not actually `pa.Scalar` (they where non-serializable `plc.Scalar` objects).